### PR TITLE
Add settings to select modes to enable

### PR DESCRIPTION
**Pull request by dtrucs**

### CODE REVIEW

File: src/modules/cameras/camera-container.js [hunk 1]
```diff
@@ -48,18 +45,36 @@ export default function(RANDO) {
...
+        .filter(([, value]) => value.enabled === true)
+        .map(([key]) => key) 
 
     /* Methods */
     RANDO.CameraContainer.prototype.init = function () {
         this._computeInitialParameters ();
-        this._buildBirdCamera ();
-        this._buildExamineCamera ();
-        this._buildHikerCamera ();
+        if (RANDO.SETTINGS.MODE.bird.enabled === true) {
```
Comment (bastyen) [inline src/modules/cameras/camera-container.js]: remove === true

File: src/modules/cameras/camera-container.js [hunk 2]
```diff
@@ -48,18 +45,36 @@ export default function(RANDO) {
...
+          this._buildBirdCamera();
+        } else {
+          this.hideCamera('bird');
+        }
+        if (RANDO.SETTINGS.MODE.examine.enabled === true) {
```
Comment (bastyen) [inline src/modules/cameras/camera-container.js]: remove === true

File: src/modules/cameras/camera-container.js [hunk 3]
```diff
@@ -48,18 +45,36 @@ export default function(RANDO) {
...
+          this._buildExamineCamera();
+        } else {
+          this.hideCamera('examine');
+        }
+        if (RANDO.SETTINGS.MODE.hiker.enabled === true) {
```
Comment (bastyen) [inline src/modules/cameras/camera-container.js]: remove === true

File: src/modules/cameras/camera-container.js [hunk 4]
```diff
@@ -48,18 +45,36 @@ export default function(RANDO) {
...
+          this._buildHikerCamera();
+        } else {
+          this.hideCamera('hiker');
+        }
+
+        if (RANDO.CameraIDs.length === 1) {
```
Comment (bastyen) [inline src/modules/cameras/camera-container.js:72]: what happens if length === 0 ?
it's better to use <= 1 and then check and handle length 0 or 1

File: src/modules/cameras/camera-container.js (hunk 1 above)
Comment (dtrucs) [reply src/modules/cameras/camera-container.js]: Fixed

File: src/modules/cameras/camera-container.js (hunk 2 above)
Comment (dtrucs) [reply src/modules/cameras/camera-container.js]: Fixed

File: src/modules/cameras/camera-container.js (hunk 3 above)
Comment (dtrucs) [reply src/modules/cameras/camera-container.js]: Fixed

File: src/modules/cameras/camera-container.js (hunk 4 above)
Comment (dtrucs) [reply src/modules/cameras/camera-container.js:72]: I fixed this problem by activating the "examine" mode if no mode is activated.

**Review by bastyen — APPROVED** [review]

**Pull request merged**